x-pack/plugins/apm/public/components/shared/charts/timeseries_chart.tsx

```diff
@@ -91,30 +64,22 @@ export function TimeseriesChart({
   yDomain,
   anomalyTimeseries,
   customTheme = {},
-}: Props) {
+  comparisonEnabled,
+  offset,
+  timeZone,
+}: TimeseriesChartProps) {
   const history = useHistory();
-  const { core } = useApmPluginContext();
   const { annotations } = useAnnotationsContext();
   const { chartRef, updatePointerEvent } = useChartPointerEventContext();
   const theme = useTheme();
   const chartTheme = useChartTheme();
```

Comment:
The abstraction work done in this PR is to make the charts usable from outside the APM context, aka not tightly coupled. 
There is a second round of work for annotations, where we could do more abstraction, but it will be in another PR.